python/lsst/sims/catUtils/mixins/snUniversalRules.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
```

Comment:
defined these. Though the doc strings could still be improved 

7fe7aee65af36b820a08da6ba2511fd9f7a84af